packages/app/src/components/Dimensions/__tests__/DimensionOptions.spec.js

```diff
@@ -37,4 +48,19 @@ describe('The DimensionOptions component ', () => {
 
         expect(dropDown.length).toEqual(1);
     });
+
+    it('passes only 1 element ("Add to Filter", or "Remove") as menuItems to <DropDown /> if prop isSelected is true and prop type is equal to YearOnYear', () => {
```

Comment:
Remove the test inside the parentheses, since this text is not actually checked. Same goes for the next test